pkg/obfuscate/sql_tokenizer.go

```diff
@@ -150,6 +150,11 @@ func (k TokenKind) String() string {
 	return str
 }
 
+const (
+	// DBMSSqlserver is a MS SQL Server
+	DBMSSqlserver = "sqlserver"
```

Comment:
```suggestion
	DBMSSQLserver = "sqlserver"
```
It doesn't seem like `sqlserver` is a valid value. I think the right one is `mssql`. Check https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#connection-level-attributes